### ISSUE TITLE
C1 Tenkeyless Keyboard not supported by Via

### ISSUE THREAD

**Issue by KawaiiBakemono** [the-via/releases#272]
**Describe the bug**
Although my keyboard is plugged in, it says "Searching for devices...", however the 'Key Tester' registers my all of my inputs.

**To Reproduce**
1. Go to 'Configure'
2. Infinitely says "Searching for devices..."

**Expected behavior**
Via should recognize my keyboard.

**Screenshots**
![image](https://github.com/the-via/releases/assets/143901494/820e3e00-e113-43ad-8610-a39effab336a)
![image](https://github.com/the-via/releases/assets/143901494/b8bf5686-370b-42b3-a6c8-d82b45651359)

**VIA Version**
- [ ] I am using https://usevia.app/
- [x] I am using the latest release of the desktop app from https://github.com/the-via/releases/releases/latest

**Keyboard firmware VIA protocol**
- Says _Requires connected device_

**Desktop (please complete the following information):**
 - OS: Kubuntu Linux 23.04

**Custom json definition?**
- [ ] I am loading a custom json definition using the designer tab

**Sanity checks**
- [x] I have tried connecting the keyboard directly to the computer (i.e. not using a USB hub)
- [ ] I have flashed the latest firmware from https://www.caniusevia.com/docs/download_firmware
- The firmware is not available for my model.

**Comment by adophoxia** [comment]
This board is not supported by VIA because it's not listed in https://www.caniusevia.com/docs/supported_keyboards.

**Comment by KawaiiBakemono** [comment]
Are there any plans to support it?

**Comment by Xelus22** [comment]
> Are there any plans to support it?

Should be asking Keychron that.

**Comment by jpjonte** [comment]
On their [official page for the C1 Pro](https://www.keychron.com/products/keychron-c1-pro-qmk-via-wired-mechanical-keyboard?variant=40615805583449), about 1/4 down the page, Keychron wrote the following:

> Since the VIA code for the C1 Pro is still awaiting approval from Github, it is not yet automatically recognized. 

Is there some kind of deadlock here with both sides waiting for a response from the other?

**Comment by Xelus22** [comment]
Idk about this but its already in... https://github.com/the-via/keyboards/pull/1912

**Comment by adophoxia** [comment]
@jpjonte, I want to be sure. You're talking about the **regular** C1, right?

**Comment by jpjonte** [comment]
@adophoxia no, I'm talking about the C1 Pro. I see now that this issue is about the C1.
But as far as I can tell that keyboard doesn't support VIA in the first place: https://www.keychron.com/products/keychron-c1-wired-mechanical-keyboard

**Comment by Philomorph** [comment]
Just picked up the C1 TKL and it's bananas that it's still not on the VIA supported list, considering the included quickstart guide in the box says to download the VIA key remapping software in multiple places.

I get the same behavior - neither the web version nor the standalone version of the software detects the keyboard, but it works fine otherwise.

**Comment by adophoxia** [comment]
> Just picked up the C1 TKL and it's bananas that it's still not on the VIA supported list, considering the included quickstart guide in the box says to download the VIA key remapping software in multiple places.
> 
> I get the same behavior - neither the web version nor the standalone version of the software detects the keyboard, but it works fine otherwise.

I like to imagine that they must've somehow copy and pasted the same information from the C Pro series in error and that's how it came to be. Still, regular C series isn't supported by VIA.

**Comment by Philomorph** [comment]
> 
> I like to imagine that they must've somehow copy and pasted the same information from the C Pro series in error and that's how it came to be. Still, regular C series isn't supported by VIA.

It's strange, because it's all over the quick reference sheet and booklet, and not in a small way. It's like they took the sheet from the Pro and just pasted a diagram of the TKL in place, leaving everything else whether it was relevant or not.

It's not a big deal - I'd maybe remap one key at most. Just annoying to go through the included instructions and then have to search online for why it doesn't work.
